docs/management/managing-indices.asciidoc

```diff
@@ -0,0 +1,22 @@
+[[managing-indices]]
+== Managing Indices
+
+include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
+
+The index management UI enables you to update your Elasticsearch indices
+either individually or in bulk. To open the UI, go to *Management* and click
+*Index Management*.
+
+[role="screenshot"]
+image::images/management-index-management.png[Index Management UI]
+
+Clicking an index name opens a panel with access to the index settings, mapping, and
+statistics. Operations for managing the index are in a context menu on the lower
+right.
+
+To perform operations on multiple indices, in the main window, select the
+checkbox for each index, or *Name* to select all indices.  The context
+menu for managing the indices appears next to the query bar.
+
```

Comment:
If you introduce the notion of "bulk operations" up above, you can probably streamline this a bit:

To perform bulk operations, go to *Index Management*, select the checkboxes of the indices you want to modify, and choose an operation from the Index options menu next to the query bar. To select all indices, select the checkbox next to the Name header.

It's "select" checkboxes, not "enable" checkboxes? Trying to file all these things away for future reference. :-)